testing/examples/run_instruction_arbitrary/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Ubuntu with bazel, gcloud and its dependencies preinstalled.
+
+FROM launcher.gcr.io/google/bazel:latest
+
+# Ensure new repos aren't compromised.
+COPY image_data/gcloud.pub.gpg /
```

Comment:
Lets substitute installing google-cloud-sdk with apt get below to instead do a wget of
https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-220.0.0-linux-x86_64.tar.gz
and install using `/google-cloud-sdk/install.sh -q`

Reply:
Done